homeassistant/components/control4/strings.json

```diff
@@ -0,0 +1,34 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "The Control4 controller associated with this account has already been configured."
+    },
+    "error": {
+      "cannot_connect": "Couldn't connect to the local controller. Check your hostname and login details.",
+      "invalid_auth": "Invalid Control4 username or password.",
+      "unknown": "An unknown error occured. Please report log info."
+    },
+    "step": {
+      "user": {
+        "data": {
+          "host": "Controller hostname",
+          "password": "Password",
```

Comment:
This should be a reference.

Probably most other strings can also be using references. Here are some references that are part of the standard template.

```python
{
                "step": {
                    "user": {
                        "data": {
                            "host": "[%key:common::config_flow::data::host%]",
                            "username": "[%key:common::config_flow::data::username%]",
                            "password": "[%key:common::config_flow::data::password%]",
                        },
                    }
                },
                "error": {
                    "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
                    "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
                    "unknown": "[%key:common::config_flow::error::unknown%]",
                },
                "abort": {
                    "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
                },
            }
```

Reply:
I tried using references and then doing `python3 -m script.translations develop` to compile the strings, but the references aren't actually replaced by the strings they're supposed to be.

For example, I'll end up seeing "[%key:common::config_flow::data::host%]" in the Home Assistant UI.

Reply:
The references will work in the production build. We haven't fixed that in the development script yet.